rootfs/templates/wrapper-body.sh

```
@@ -0,0 +1,578 @@
+			CONTAINER_ID=$(docker run --detach --init --name "${DOCKER_NAME}" "${DOCKER_ARGS[@]}" ${GEODESIC_DOCKER_EXTRA_ARGS} ${DOCKER_IMAGE} /usr/local/sbin/shell-monitor)
+			echo "# Started session ${CONTAINER_ID:0:12}. Starting shell via \`docker exec\`..."
+			docker exec -it --detach-keys "ctrl-^,ctrl-[,ctrl-@" --env GEODESIC_HOST_CWD="${GEODESIC_HOST_CWD}" "${DOCKER_NAME}" /bin/bash -l $*
+		fi
+	fi
+	true
+}
+
+_polite_stop() {
+	name="$1"
+	[ -n "$name" ] || return 1
+	if [ $(docker ps -q --filter "name=${name}" | wc -l | tr -d " ") -eq 0 ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)